src/Umbraco.Web.UI.Client/src/common/directives/components/forms/umbcheckbox.directive.js

```diff
@@ -91,6 +91,7 @@
             cssClass: "@?",
             iconClass: "@?", // deprecated
             icon: "@?",
+            svg: "@?",
```

Comment:
@RachBreeze it seems you forgot to remove `svg` added in the component here.

Reply:
Oops!  Sorry on it